content/v2.0/query-data/get-started/syntax-basics.md

```diff
@@ -61,10 +61,10 @@ this is a string
 Flux also supports objects. Each value in an object can be a different data type.
 
 ```js
-> o = {name:"Jim", age: 42}
+> o = {name:"Jim", age: 42, "favorite color": "red"}
```

Comment:
Can favorite color have quotes bc it's 2 words--or does that only work w bracket notation?

Reply:
This is how you define an object. Object property keys that have special characters or whitespace characters must be wrapped in double quotes.